src/style.css

```diff
@@ -1,3 +1,80 @@
+html {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
 body {
   margin: 0;
```

Comment:
CSS: Don't use * selector (it impacts performance). Set styles only for elements that require them. Zeroing out your margins, paddings or other styles with * is still inefficient for browser.